browser/ui/views/side_panel/ai_chat/ai_chat_side_panel_web_view.cc

```diff
@@ -66,3 +73,35 @@ void AIChatSidePanelWebView::OnShow() {
     should_focus_ = false;
   }
 }
+
+content::WebContents* AIChatSidePanelWebView::AddNewContents(
+    content::WebContents* source,
+    std::unique_ptr<content::WebContents> new_contents,
+    const GURL& target_url,
+    WindowOpenDisposition disposition,
+    const blink::mojom::WindowFeatures& window_features,
+    bool user_gesture,
+    bool* was_blocked) {
+  auto* browser_view = BrowserView::GetBrowserViewForNativeWindow(
+      this->GetWidget()->GetNativeWindow());
+  auto* browser = browser_view->browser();
+
+  // If AI Chat is not open in the side panel, don't open the tab.
+  if (browser->browser_window_features()
```

Comment:
@sangwoo108 is there a better way to get to the browser?

Reply:
I think this is the best we can so far.